src/main/java/wooteco/subway/line/repository/LineJdbcDao.java

```diff
@@ -0,0 +1,71 @@
+package wooteco.subway.line.repository;
+
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import wooteco.subway.line.Line;
+
+import java.sql.PreparedStatement;
+import java.util.List;
+import java.util.Optional;
+
+public class LineJdbcDao implements LineRepository {
```

Comment:
```suggestion
public class JdbcLineDao implements LineDao {
```
보통 Jdbc, InMemory 같은 세부사항을 맨 앞에 표현해줘요. 팀 컨벤션마다 차이가 있으니 참고하세요